models/classes/Security/DataAccess/Repository/CachedPlatformKeyChainRepository.php

```diff
@@ -51,6 +49,10 @@ public function save(KeyChain $keyChain): void
 
     public function findAll(KeyChainQuery $query): KeyChainCollection
     {
+        if ($query->getIdentifier() == null) {
+            $query = new KeyChainQuery($this->getOption(PlatformKeyChainRepository::OPTION_DEFAULT_KEY_ID));
```

Comment:
I think you are getting the option from a different class here, so you should use `PlatformKeyChainRepository::getOption()`, no?